ga4gh/repo_manager.py

```diff
@@ -383,6 +400,45 @@ def removeReferenceSet(self, referenceSetName):
         self._repoEmit("ReferenceSet '{}' removed".format(
             referenceSetName))
 
+    def addBioSample(self, datasetName, filePath, moveMode):
+        """
+        Add a BioSample
+        """
+        # move the bam file
+        self._check()
+        self._checkDataset(datasetName)
+        fileName = os.path.basename(filePath)
+        destPath = os.path.join(
+            self._repoPath, self.datasetsDirName, datasetName,
+            self.bioSamplesDirName)
+        fullDest = os.path.join(destPath, fileName)
+        self._checkFile(filePath, self.jsonExtension)
+        dataset = datasets.AbstractDataset('dataset1')
```

Comment:
I would choose a name other than `dataset1` to emphasize this is just a throwaway dataset